### ISSUE TITLE
Bug: ...

### ISSUE THREAD

**Issue by Frepke** [qdm12/private-internet-access-docker#176]
**TLDR**: *Running the container with DOT=off, my ISP DNS addresses are used instead of the pushed DNS addresses from the vpn provider*

1. Is this urgent?

    - [x] Yes
    - [ ] No

2. What VPN service provider are you using?

    - [ ] PIA
    - [ ] Mullvad
    - [ ] Windscribe
    - [x] Surfshark
    - [ ] Cyberghost

3. What's the version of the program?

    **See the line at the top of your logs**

    `Running version latest built on 2020-06-18T18:06:34Z (commit cdbb7bf)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

With the container running with DOT=off, dns requests are leaking.
When running openvpn straight from the host "sudo openvpn us-dal.prod.surfshark.com_udp.ovpn" there's no leak. 

Logs:

```log

```

Configuration file:

```yml

```

Host OS: Linux

**Comment by qdm12** [comment]
Hello! I added `script-security 2` to the openvpn configuration files, so it should now be able to run whatever script is pushed by the vpn provider. Let me know if it works. Thanks!

**Comment by Frepke** [comment]
Thanks Quentin,

I tested the latest build with DOT=off, but my ISP DNS is still leaking. I did also a test with PIA, but with the same result.

Maybe an ENV for manual DNS addresses when DOT=off is an option.

**Comment by Frepke** [comment]
With the container running with DOT=off, dns requests are leaking.
When running openvpn straight from the host "sudo openvpn us-dal.prod.surfshark.com_udp.ovpn" there's no leak.

I did some tests on a different systems with "sudo openvpn us-dal.prod.surfshark.com_udp.ovpn", their DNS addresses aren't pushed (or at least not always). When the DNS addresses are filled in manually, then the leak is gone.

**Comment by Frepke** [comment]
Extra info.
I followed this link for my manual openvpn setup: https://support.surfshark.com/hc/en-us/articles/360013425373-How-to-set-up-Surfshark-VPN-on-Raspberry-Pi. In this tutorial there's also a link to the .ovpn files you can use.
With this setup there's still a dns leak, but when I setup the DNS addresses manually on the host (DNS 1 = 162.252.172.57
DNS 2 = 149.154.159.92) the leak is gone.
